huaweicloud/services/vpc/resource_huaweicloud_vpc.go

```diff
@@ -201,26 +202,45 @@ func resourceVirtualPrivateCloudRead(_ context.Context, d *schema.ResourceData,
 	d.Set("routes", routes)
 
 	// save VirtualPrivateCloudV2 tags
-	if vpcV2Client, err := config.NetworkingV2Client(config.GetRegion(d)); err == nil {
-		if resourceTags, err := tags.Get(vpcV2Client, "vpcs", d.Id()).Extract(); err == nil {
-			tagmap := utils.TagsToMap(resourceTags.Tags)
-			if err := d.Set("tags", tagmap); err != nil {
-				return diag.Errorf("error saving tags to state for VPC (%s): %s", d.Id(), err)
-			}
-		} else {
-			log.Printf("[WARN] Error fetching tags of VPC (%s): %s", d.Id(), err)
-		}
-	} else {
-		return diag.Errorf("error creating VPC client: %s", err)
+	v2Client, err := conf.NetworkingV2Client(region)
+	if err != nil {
+		return diag.Errorf("error creating VPC v2 client: %s", err)
+	}
+
+	resourceTags, err := tags.Get(v2Client, "vpcs", d.Id()).Extract()
+	if err != nil {
+		return diag.Errorf("error fetching tags of VPC (%s): %s", d.Id(), err)
```

Comment:
it's not suggested to change the logical when adding a new feature.

Reply:
It was changed because of LINT issues. Only the way it is written has changed